winpr/libwinpr/crt/unicode.c

```diff
@@ -30,7 +30,15 @@
 
 #ifndef _WIN32
 
+#if defined(WITH_ICU)
```

Comment:
I think instead of all these ugly #ifdefs we can do it in a cleaner way. Why not provide a separate **unicode_icu.c** file with ICU-based implementations for all the functions from <winpr/string.h>?